packages/components/src/site-thumbnail/index.tsx

```diff
@@ -1,45 +1,70 @@
+import { css, CSSObject } from '@emotion/css';
 import classnames from 'classnames';
 import { ReactNode } from 'react';
 import './style.scss';
 import { MShotsOptions, useMshotsImg } from './use-mshots-img';
 import { getTextColorFromBackground } from './utils';
 
-const MSHOTS_OPTION: MShotsOptions = {
-	vpw: 1200,
-	vph: 1200,
-	w: 374 * 2,
+type SizeCss = CSSObject & { width: number; height: number; sizes?: string };
+const SIZES: { [ sizeName: string ]: SizeCss } = {
+	small: {
+		width: 108,
+		height: 78,
+	},
+	medium: {
+		width: 401,
+		height: 401 * ( 11 / 16 ),
+		sizes: [
```

Comment:
`sizes` will always need to be passed in by the caller, because it's contextual to how the image is being displayed (i.e. 1/2 width of a page vs. a part of a 4 column grid).

Reply:
I'm happy to work on this and refactor it.
But I don't know how much it's worth it for the user. Probably having just the basic image, plus 2x and 3x is more than enough. Doing that we can remove the sizes array.

Reply:
BTW, this will be the only place where we are using `srcset` and `sizes` 😍 .

Reply:
I don't know if I should feel proud or sad with that information, @sejas 🤣

Reply:
@danielbachhuber Now SiteThumbnail accepts an object with the actual size instead of one string.
I preferred to keep a some dimensions as contants inside the component https://github.com/Automattic/wp-calypso/pull/66988/files#diff-8e30ead238a766d262e6016162eebd9be8d5110bc2c6243f41ffc715d792bb19R9-R21

And the img `sizes` attribute out of the component:
https://github.com/Automattic/wp-calypso/pull/66988/files#diff-6c8c71821230e21d1cc72b907696300b034195bbb986fa5cd3bcbb591985b44fR20-R26


I've also added some tests 💪 
https://github.com/Automattic/wp-calypso/pull/66988/files#diff-f638976651dd414b09a526f00a5ecae87b0077807f1e45635eca5c4677140ffe